optaweb-employee-rostering-standalone/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+quarkus.package.type=fast-jar
+
+# Swagger UI
+quarkus.swagger-ui.always-include=true
+quarkus.swagger-ui.path=/swagger-ui.html
+
+# Datasource
+# - using an embedded in-memory DB: http://h2database.com/html/features.html#in_memory_databases
+# - not closing the DB automatically: http://h2database.com/html/features.html#closing_a_database
+quarkus.datasource.db-kind=h2
```

Comment:
Ignore the previous comment. I had some pending comments and I couldn't find them, so I submitted the review.

PostgreSQL is configured using the `%postgres` profile - that's OK.

Reply:
We do want to keep the structure of the OptaWebs one-to-one if possible. Which means either adding `optaweb-employee-rostering-tests` (which defaults to h2) and change the default here to postgres, or modify `optaweb-vehicle-routing` to use this approach. It feel odd saying "run with the JAR from `optaweb-employee-rostering-tests" in the README, so I probably will add a `runLocally.sh` that does that. (The command changes either way since I am not using the legacy JAR)

Reply:
It's a PITA that we can't use the same JAR with different DBs with Quarkus. Will we end up with `standalone-postgres` and `standalone-h2` and switch between them depending on whether you `runLocally` or `runOnOpenShift` or based on a `--database` switch? Or can we only have a single `standalone` module with different profiles for PostgreSQL and H2 and rebuild it each time we `runLocally` or `runOnOpenShift`?

Is it OK to only run ITs against H2 or should we also cover the PostgreSQL configuration?

I think we need to figure out all of that but we can do that later.